src/Riok.Mapperly/Emit/Syntax/SyntaxFactoryHelper.SymbolDeclaration.cs

```diff
@@ -17,13 +17,19 @@ public NamespaceDeclarationSyntax Namespace(string ns)
 
     public ClassDeclarationSyntax Class(string name, SyntaxTokenList modifiers, SyntaxList<MemberDeclarationSyntax> members)
     {
+        var isVisible = modifiers.Any(
```

Comment:
Instead of checking whether it is public, wouldn't it make more sense to check whether it is not partial and add the generated code attribute to all non-partial classes? A partial class usually contains content at other locations which we can't be sure of that it is generated code (it usually is the declaration to the generated code...). All non partial classes are completely generated by Mapperly.

Reply:
> wouldn't it make more sense to check whether it is not partial
Of course it would... :) 

Also addressed the other suggestions.